### ISSUE TITLE
High CPU usage (linux installed via flatpak/snapd)

### ISSUE THREAD

**Issue by cspwizard** [jgraph/drawio-desktop#1927]
### Preflight Checklist

* [ x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio-desktop/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [ x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Describe the bug**
High CPU usage of draw.io app in linux. It consumes more that 50% even nothing is happening. Simply opened the chard and leave it idle in background.

![image](https://github.com/user-attachments/assets/c33fd97d-c7e3-4107-a0d3-589108f02e49)

chart isn't small, but still...
![image](https://github.com/user-attachments/assets/79f7064a-effa-45b3-ad6a-70f0bf5fa9da)

**To Reproduce**
1. Create chart with a big number of elements
2. Open it
3. Do nothing
4. See high CPU usage

**Expected behavior**
CPU should be idle, when nothing is going on in the app 

**Screenshots**

**draw.io version (In the Help->About menu of the draw.io editor):**

- draw.io version 25.0.1 (also reproduced on previous versions)

**Desktop (please complete the following information):**
 - OS: Linux (Xubuntu 24.10)

**Additional context**

**Comment by davidjgraph** [comment]
Does this still happen with 26.2.15?

**Comment by cspwizard** [comment]
yep, still ~35-50% cpu usage while idle, also periodically spikes IO

**Comment by davidjgraph** [comment]
Valid as this might be, it appears to be boundary case and we cannot justify the effort tracking down such a case, apologies.

**Comment by cspwizard** [comment]
is there any way I can help with debug/looking for root cause?
